actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -534,6 +543,10 @@ def digest
         def key_generator
           request.key_generator
         end
+
+        def same_site_protection
```

Comment:
It's not but I followed the pattern which is to have access to this as part of the cookie. Want me to remove it?